### PR TITLE
MARKET-1646 Rename some event handlers

### PR DESCRIPTION
**Pull request by StasivPavlo**

### CODE REVIEW

File: components/bl-carousel-component/src/index.js [hunk 1]
```diff
@@ -102,8 +102,8 @@ export default function Carousel({ component, eventHandlers }) {
       onMouseLeave={ onMouseLeave }>
       { withControls && (
         <div className="carousel__controls-button">
-          <PrevButton disabled={ !!animation } onPrevButton={ onPrevButton }/>
-          <NextButton disabled={ !!animation } onNextButton={ onNextButton }/>
+          <PrevButton disabled={ !!animation } onPrevButtonClick={ onPrevButtonClick }/>
```
Comment (v-excelsior) [inline components/bl-carousel-component/src/index.js:105]: Component with name PrevButton mustn't know about onPrevButtonClick action. He should know what to do when onClick triggered.
```suggestion
          <PrevButton disabled={ !!animation } onClick={ onPrevButtonClick }/>
```

File: components/bl-carousel-component/src/subcomponents.js [hunk 2]
```diff
@@ -98,24 +98,24 @@ export function CarouselIndicators(props) {
   );
 }
 
-export function NextButton({ disabled, onNextButton }) {
+export function NextButton({ disabled, onNextButtonClick }) {
```
Comment (v-excelsior) [inline components/bl-carousel-component/src/subcomponents.js:101]: As we already here. Please, refactor that to one component Button which accept type and icon component as props.

**Review by v-excelsior — CHANGES_REQUESTED** [review]

File: components/bl-carousel-component/src/subcomponents.js [hunk 3]
```diff
@@ -98,38 +98,26 @@ export function CarouselIndicators(props) {
   );
 }
 
-export function NextButton({ disabled, onNextButton }) {
+export function Button({ disabled, onClick, type, Icon }) {
```
Comment (v-excelsior) [inline components/bl-carousel-component/src/subcomponents.js:101]: We can avoid passing Icon, as we already pass the type.
Just use the condition to choose the icon inside the component.

Also, I do not clearly understand why we don't use disable property on the button but use disabling pointer events through styles.

Comment (StasivPavlo) [reply components/bl-carousel-component/src/subcomponents.js]: At first, I was using disable, but later I found a bug that @Valodya helped me solve with pointer events

**Pull request merged**